### ISSUE TITLE
Timestamp Formatting

### ISSUE THREAD

**Issue by flapjackfritz** [phallguy/shog#1]
How would I go about formatting the timestamp?  Is it something I could do in config?

**Comment by phallguy** [comment]
There's no way to customize the format of the time right now, only the presence. It'd be easy enough to adjust the Formatter#timestamp method to take a proc that could be used to parse and format the time.

**Comment by phallguy** [comment]
Did a little digging, and the API for logging allows somewhat arbitrary formatting of the timestamps so it'd be really hard to parse it reliably for reformatting. I'm going to close this as I don't think custom timestamp formatting would be a good fit for shog at the moment.
